Add documentation for the excludeResourcePrefixes (#2328)
kubernetes-sigs/kueue: site/content/en/docs/tasks/manage/administer_cluster_quotas.md

```shell
kubectl apply -f team-a-cq.yaml -f team-b-cq.yaml -f shared-cq.yaml
```

## Exclude arbitrary resources in the quota management 

> By default, administrators must specify all resources used by any Pods in the ClusterQueues `.spec.resourceGroups[*]`.

Reviewer: ```suggestion
By default, administrators must specify all resources required by Pods in the ClusterQueues `.spec.resourceGroups[*]`.
```

Author: Done.

> If you want to exclude some resources for the specific devices in the ClusterQueues management, 

Reviewer: ```suggestion
If you want to exclude some resources in the ClusterQueues quota management and admission process, 
```

Author: Done.

> you can specify the resource prefixes in the Kueue configuration as a cluster-level setting.

Reviewer: ```suggestion
you can specify the resource prefixes in the Kueue Configuration as a cluster-level setting.
```

Author: Done.

> Follow the instructions described
> [here](/docs/installation#install-a-custom-configured-released-version) to
> install a release version by extending the configuration with the following
> fields:

Reviewer: ```suggestion
Follow the [installation instructions for using a custom configuration](/docs/installation#install-a-custom-configured-released-version)
and extend the configuration with fields similar to the following:
```

Author: Done.

> ```yaml

Reviewer: ```suggestion
```yaml
apiVersion: config.kueue.x-k8s.io/v1beta1
kind: Configuration
```

Author: Done.
